packages/client/src/links/httpBatchLink.ts

```diff
@@ -1,14 +1,20 @@
 import { AnyRouter, ProcedureType } from '@trpc/server';
 import { TRPCResponse } from '@trpc/server/rpc';
+import { getFetch } from '..';
```

Comment:
```suggestion
import { getFetch } from '../getFetch';
```
better for other runtimes (Deno) and code run without tree shaking